rustls/src/quic.rs

```diff
@@ -108,8 +164,78 @@ impl PacketKey {
                 &[],
             )),
             iv: hkdf_expand(secret, IvLen, b"quic iv", &[]),
+            suite,
         }
     }
+
+    /// Encrypt a QUIC packet
+    ///
+    /// Takes a header (which is used as the additional authenticated data) and payload. The
+    /// final `tag_len()` bytes of the payload will be overwritten with the authentication tag.
```

Comment:
The sentence "The final ..." is now inaccurate. It should state that the authentication tag is returned.

Reply:
Fixed.